### PR TITLE
Grapple hook initial implementation

### PR DESCRIPTION
**Pull request by adaless5**
Created a "Tool" base which both base equipment and base weapon will inherit from. This made the Tool_Belt system a bit easier. Also implemented a temporary item pick up to test "Obtaining Equipment"

### CODE REVIEW

File: CAPSTONE/Assets/Programming/Scripts/GrappleHook.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GrappleHook : Equipment
+{
+    public enum PlayerState
```
Comment (DrApok) [inline CAPSTONE/Assets/Programming/Scripts/GrappleHook.cs:7]: This should be in the Player controller class.

**Review by DrApok — CHANGES_REQUESTED** [review]
This looks great bro! Could you move the player state over to the player controller class to keep that functionality encapsulated.

**Review by DrApok — APPROVED** [review]
Awesome bro!

**Review by NickGhezzi — APPROVED** [review]
i love you

**Pull request merged**